CHANGELOG.md

```diff
@@ -1,18 +1,66 @@
 # App Center SDK for .NET Change Log
 
-## Version 2.2.0 (Under development)
+## Version 2.2.0-preview (Under development)
+
+This preview version adds support for WPF and WinForms applications for the Analytics and Crashes modules.
+
+This version temporarily removes support for UWP for Crashes as long as the package is in preview (the new UWP SDK implementation sends error reports without using the Windows system reporting but the AppCenter backend is not yet ready to process the data).
```

Comment:
```suggestion
This preview version removes Crashes support for UWP. It will return in the future, but the App Center backend is not yet ready to process UWP crashes using the system introduced in this version. If your app relies on UWP crashes, do not update to this preview version.
```

Reply:
I don't understand the "it will return" part.

Reply:
I don't understand the "it will return" part especially with the "but" after.

Reply:
I clarified it

Reply:
PR review reviews 😆